src/store/singing.ts

```diff
@@ -95,26 +91,34 @@ const generateNoteEvents = (notes: Note[], tempos: Tempo[], tpqn: number) => {
 
 let audioContext: AudioContext | undefined;
 let transport: Transport | undefined;
-let previewSynth: PolySynth | undefined;
-let channelStrip: ChannelStrip | undefined;
+let globalChannelStrip: ChannelStrip | undefined;
+const trackNodes: TrackNode[] = [];
```

Comment:
`previewSynth`は以前と同じようにここで持って、
`trackNodes`は`channelStrips`にして`TrackId`をキーにMapで保持が良いかも。
```suggestion
let previewSynth: PolySynth | undefined; // ノート編集時・鍵盤を弾いたときの音のプレビュー用
let globalChannelStrip: ChannelStrip | undefined;
const channelStrips = new Map<TrackId, ChannelStrip>();
```